lib/stacks/eks-blueprint-stack.ts

```diff
@@ -54,9 +53,16 @@ export class EksBlueprintProps {
      * Control Plane log types to be enabled (if not passed, none)
      * If wrong types are included, will throw an error.
      */
-    readonly enableControlPlaneLogTypes?: string[];
+    readonly enableControlPlaneLogTypes?: enabledControlPlaneLogTypes[];
 }
 
+export enum enabledControlPlaneLogTypes {
```

Comment:
enum is a data type. Should follow the noun semantics. It does NOT represent enabled log types, they are just available control plane log types. So should be ControlPlaneLogTypes (starting with upper case). 

For more information on naming conventions please read: https://google.github.io/styleguide/tsguide.html

Reply:
That is a very helpful link I will be sure to reference it in the future. Also I have fixed that now.